WORKSPACE

```diff
@@ -1,18 +1,58 @@
-# proto_library, cc_proto_library, and java_proto_library rules implicitly
-# depend on @com_google_protobuf for protoc and proto runtimes.
-# This statement defines the @com_google_protobuf repo.
+workspace(name = "com_github_cgrushko_proto_library")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# rules_cc defines rules for generating C++ code from Protocol Buffers.
+http_archive(
+    name = "rules_cc",
+    sha256 = "35f2fb4ea0b3e61ad64a369de284e4fbbdcdba71836a5555abb5e194cf119509",
+    strip_prefix = "rules_cc-624b5d59dfb45672d4239422fa1e3de1822ee110",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_cc/archive/624b5d59dfb45672d4239422fa1e3de1822ee110.tar.gz",
+        "https://github.com/bazelbuild/rules_cc/archive/624b5d59dfb45672d4239422fa1e3de1822ee110.tar.gz",
```

Comment:
can you point to a specific release, rather than a commit?
commits are kind of random. Also, the sha256 above usually goes out of sync because these commit-zips are generated on the fly and are not the same every time.

Reply:
There are no releases for `rules_cc` and `rules_proto`, and the latest release of Protobuf misses some fixes for `--incompatible_load_...`.

Reply:
Got it. The sha256 will be incorrect probably at some point, so might be worth commenting it out and explaining this.